src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    public static void doWork() {
+        System.out.println("Bulldozer started its work");
+
+    }
+
+    public static void stopWork() {
```

Comment:
Similar to 'doWork', the method 'stopWork' should also not be static for the same reasons mentioned above.